docs/toml.md

```diff
@@ -551,14 +551,17 @@ address = ":8080"
 # To enable basic auth on the webui
 # with 2 user/pass: test:test and test2:test2
 # Passwords can be encoded in MD5, SHA1 and BCrypt: you can use htpasswd to generate those ones
+# Users can be specified directly, or indirectly by referencing a file; if both are provided, the two are merged, with file contents having precedence
```

Comment:
> By direct, you mean auth information embedded in the request?

I'm not sure what request you are referring to. This is configuration. The users can be specified directly in the toml (as is done currently) _or_ (after this PR) in a file, which you reference in the toml (i.e. indirectly).